var/spack/repos/builtin/packages/xfd/package.py

```diff
@@ -25,3 +27,20 @@ class Xfd(AutotoolsPackage):
     depends_on('xproto@7.0.17:', type='build')
     depends_on('pkgconfig', type='build')
     depends_on('util-macros', type='build')
+
+    # Xfd requires libintl (gettext), but does not test for it
+    # correctly, so add it here.
```

Comment:
To be clear: are you saying that while `xfd` supports a configure option like `--with-gettext=...`, that it isn't handled properly?